pkg/assets/config.go

```diff
@@ -43,6 +46,7 @@ type KubeAPIServerSecretsConfig struct {
 type KubeAPIServerConfigMapsConfig struct {
 	Namespace string
 
+	Config              []byte
```

Comment:
make this more explicit for what this is?

Reply:
done